demisto_sdk/commands/common/constants.py

```diff
@@ -513,6 +510,26 @@ def found_hidden_param(parameter_name):
 
 INTEGRATION_CHANGELOG_REGEX = r'{}{}/([^\\/]+)/CHANGELOG.md$'.format(CAN_START_WITH_DOT_SLASH, INTEGRATIONS_DIR)
 
+PACKS_REGEX = r'{}{}\/([^\\\/]+)'.format(CAN_START_WITH_DOT_SLASH, PACKS_DIR)
+INTEGRATIONS_DIR_REGEX = r'{}\/{}'.format(PACKS_REGEX, INTEGRATIONS_DIR)
+SCRIPTS_DIR_REGEX = r'{}\/{}'.format(PACKS_DIR, SCRIPTS_DIR)
+LAYOUTS_DIR_REGEX = r'{}\/{}'.format(PACKS_DIR, LAYOUTS_DIR)
+WIDGETS_DIR_REGEX = r'{}\/{}'.format(PACKS_DIR, WIDGETS_DIR)
+DASHBOARDS_DIR_REGEX = r'{}\/{}'.format(PACKS_DIR, DASHBOARDS_DIR)
+REPORTS_DIR_REGEX = r'{}\/{}'.format(PACKS_DIR, REPORTS_DIR)
+RELEASE_NOTES_DIR_REGEX = r'{}\/{}'.format(PACKS_DIR, RELEASE_NOTES_DIR)
+INCIDENT_TYPES_DIR_REGEX = r'{}\/{}'.format(PACKS_DIR, INCIDENT_TYPES_DIR)
+INCIDENT_FIELDS_DIR_REGEX = r'{}\/{}'.format(PACKS_DIR, INCIDENT_FIELDS_DIR)
+INDICATOR_TYPES_DIR_REGEX = r'{}\/{}'.format(PACKS_DIR, INDICATOR_TYPES_DIR)
+INDICATOR_FIELDS_DIR_REGEX = r'{}\/{}'.format(PACKS_DIR, INDICATOR_FIELDS_DIR)
+CONNECTIONS_DIR_REGEX = r'{}\/{}'.format(PACKS_REGEX, CONNECTIONS_DIR)
+
+PLAYBOOKS_DIR_REGEX = r'{}\/{}'.format(PACKS_REGEX, PLAYBOOKS_DIR)
+PLAYBOOK_YML_REGEX = r'{}\/.*\.yml'.format(PLAYBOOKS_DIR_REGEX)
+PLAYBOOK_CHANGELOG_REGEX = r'{}\/.*_CHANGELOG\.md$'.format(PLAYBOOKS_DIR_REGEX)
+
+TEST_PLAYBOOKS_DIR_REGEX = r'{}\/{}'.format(CONNECTIONS_DIR, TEST_PLAYBOOKS_DIR)
```

Comment:
Where are we using them all?

Reply:
We are not using them yet. But this is the base to replace the existing regexes.
It will require more work to replace all the existing regexes